bindata/etcd/pod.yaml

```diff
@@ -23,7 +23,9 @@ spec:
         - |
           #!/bin/sh
           echo -n "Fixing etcd log permissions."
-          mkdir -p /var/log/etcd && chmod 0700 /var/log/etcd
+          mkdir -p /var/log/etcd  && chmod 0700 /var/log/etcd
+          echo -n "Fixing etcd auto backup permissions."
+          mkdir -p /var/lib/etcd-auto-backup  && chmod 0700 /var/lib/etcd-auto-backup
```

Comment:
why set the executable bit?

Reply:
I actually wanted to make it `600` but i followed the same permission as for log dir

Reply:
fixed 👍🏽

Reply:
I have some doubts that whoever is restoring from that backup is the exact same user that's running etcd, but okay, we can fix it later.

Reply:
ah u mean the `/var/log/etcd` .. ok i can change it now, shall i ?

Reply:
raised https://github.com/openshift/cluster-etcd-operator/pull/1333 for the log's permission